src/sql/workbench/contrib/welcome/page/browser/media/close.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10 10" width="10" height="10"><title>close_small</title><path d="M5.58,5.11,10,9.53l-.58.58L5,5.69.58,10.11,0,9.53,4.42,5.11,0,.69.58.11,5,4.53,9.42.11,10,.69Z"/></svg>
```

Comment:
I assume close is just the X in the top right? VS Code already has this - you can just use the `codicon codicon-close` classes to utilize theirs instead of making another one

Reply:
I believe this has been fixed